andela/ah-kgl-avengers-frontend
# #166495422 Create a home page

### What does this PR do?
Setting up a sample home page that has `navbar`, `navigation` and `footer` components
### Description of Task to be completed
- Removing the existing dummy components template.
- Creating a new welcome page based on the mockup we have.
- Creating other reusable components like `footer` and `navigation header`.
- Writing unit tests of those components.
### How should this be manually tested?
- Clone [repo](https://github.com/andela/ah-kgl-avengers-frontend/) and checkout to branch `ft-home-page-166495422`.
- Install dependencies by running `npm install` script.
- Run `npm start` script to run the application in the browser.
- Run `npm test` to check all test.
### Any background context you want to provide?
- The application runs on port `8000` make sure no other process runs on the same port.
- Home page body has no content. Navigation and Footer, one is underneath another but when articles loaded in Home component the footer will span down. 
### What are the relevant pivotal tracker stories?
[#166495422](https://www.pivotaltracker.com/n/projects/2321840/stories/166495422)
### Screenshots (if appropriate)
**Full screen**
![image](https://user-images.githubusercontent.com/29593858/59333971-c0bf0400-8cf9-11e9-893a-46132ceb05bb.png)
**Mobile version**
<img src="https://user-images.githubusercontent.com/29593858/59333982-c6b4e500-8cf9-11e9-9f4c-70fcc43f9ea5.png" width="320px">
### Questions:
N/A
